cpp/src/ray/runtime/object/native_object_store.cc

```diff
@@ -73,11 +74,15 @@ std::vector<std::shared_ptr<msgpack::sbuffer>> NativeObjectStore::GetRaw(
   result_sbuffers.reserve(results.size());
   for (size_t i = 0; i < results.size(); i++) {
     auto meta = results[i]->GetMetadata();
+    auto data_buffer = results[i]->GetData();
     if (meta != nullptr) {
+      bool empty = data_buffer->Size() == 0;
+      std::string data_str =
+          empty ? "" : std::string((char *)data_buffer->Data(), data_buffer->Size());
```

Comment:
The `data_str` local variable makes a copy of the data buffer and makes the performance bad. We should avoid this.

Reply:
done.